packages/app/src/systems/Home/routes.tsx

```diff
@@ -10,7 +12,10 @@ export const homeRoutes = (
     element={
       <PrivateRoute>
         <Home />
+        <Outlet />
       </PrivateRoute>
     }
-  />
+  >
+    <Route path={Pages.faucet} element={<FaucetDialog />} />
```

Comment:
The modal here is with almost the entire width of the screen, which should be good to decrease it according to the Figma layout :)

<img width="300" alt="CleanShot 2022-09-15 at 23 05 55@2x" src="https://user-images.githubusercontent.com/2029172/190568304-f348a435-a31c-4d22-a1cf-b9dadfdc62fc.png">

Reply:
skipping this because it's not possible to resize captcha iframe and get the size we want